index.js

```diff
@@ -18,4 +18,15 @@ if (args.test || args.t) {
     console.error('You need to use argument "--test" or "--approve"');
 }
 
-launchBackstop(commandToRun, projectName);
+async function run() {
+  try {
+    const availableLocalHost = await checkLocalHost(projectName);
+    if (availableLocalHost === true) {
```

Comment:
тут так же не нужно сравнивать с тру, если ты в резолв передаешь булиан
```suggestion
    if (availableLocalHost) {
```